crates/holochain_types/src/app.rs

```diff
@@ -270,7 +270,7 @@ pub enum RoleSettingsYaml {
 #[serde(tag = "type", content = "value", rename_all = "snake_case")]
 pub enum AppBundleSource {
     /// The actual serialized bytes of a bundle
```

Comment:
What is meant by "serialized" here? This is just the raw bytes of a .happ file right?